README.md

```diff
@@ -1,17 +1,45 @@
-# Deepx's Challenge
+# DeepX Challenge
 
-Hey! We want to take a look at your code skills, choose one of the problems and do your best!
+Hey! We want to take a look at your coding skills.
 
-## Important notes
+We have different challenges and you can select the one you feel you are going to do the best in.
+Each different challenge has its own description, requirements and may have specifications on what will also be evaluated from your code.
 
-* Pick whatever challenge you feel more comfortable with from the list bellow.
-* Please organize, design, test, document your code as if it were going into production, then send us a link to the hosted repository (e.g. Github, Bitbucket...).
-* Any questions just reach out to us and we will gladly help.
-* Take as much time as needed and communicate as much as necessary.
-* Be aware that everything you do will be analyzed.
+Read everything carefully and please ask any questions you want about the evaluation or the challenges.
+
+## What will overall be evaluated from your challenge resolution
+
+Your problem solving skills, abilites with Git, the languages and frameworks selected to solve the challenges and your decision making skills will be evaluated. We also expect you to be able to deliver good quality code and your code will be evaluated in some specific aspects:
+
+1. Static Analysis:
+    - Code Quality
+    - Standards conformity
+    - Branching, Complexity and Data Flow
+2. Semantic Analysis:
+    - Deliverability
+    - Testability
+    - Coherence
+    - Cohesion
+    - Coupling
+
+Everything you do will be evaluated.
```

Comment:
this contradicts the previous statement.. either the evaluation is specific or general

Reply:
leftover actually heh